translations/en_GB/atrium-translations-en_GB-common/src/main/kotlin/ch/tutteli/atrium/translations/DescriptionMapAssertion.kt

```diff
@@ -8,5 +8,6 @@ import ch.tutteli.atrium.reporting.translating.StringBasedTranslatable
  */
 enum class DescriptionMapAssertion(override val value: String) : StringBasedTranslatable {
     CANNOT_EVALUATE_KEY_DOES_NOT_EXIST("$COULD_NOT_EVALUATE_DEFINED_ASSERTIONS -- given key does not exist.\n$VISIT_COULD_NOT_EVALUATE_ASSERTIONS"),
-    KEY_DOES_NOT_EXIST("❗❗ key does not exist")
+    KEY_DOES_NOT_EXIST("❗❗ key does not exist"),
+    MAP_CONTAINS("Map contains key")
```

Comment:
Do not forget to add your translation to de_CH => search for second DescriptionMapAssertion. There it should be:
`CONTAINS_KEY("enthält Key")